src/databases/VTK/avtPVDFileReader.C

```diff
@@ -114,11 +114,11 @@ avtPVDFileReader::ReadInFile(int _domain)
         // parse the file
         vtkNew<vtkXMLDataParser> parser;
 
-        string baseDir = FileFunctions::Dirname(string(filename)) +
+        string baseDir = FileFunctions::Dirname(filename) +
```

Comment:
Is it worth adding any modification comments to the function preamble? I have no opinion one way or the other. Just noticed that there were none.